versioned_docs/version-1.10/cloud/preview-environments/dashboard.mdx

```diff
@@ -7,6 +7,9 @@ id: dashboard
 
 import Image from '@theme/Image';
 
+:::caution
+This feature is only available on Okteto Cloud, a product that has been deprecated. These instructions do not apply to, and will not work for, SaaS or Self-Hosted customers. Please us the [GitHub Actions](/cloud/preview-environments/preview-environments-github/) instructions instead to setup Preview Environments in Okteto. This page will soon be deprecated as well.
```

Comment:
We haven't made the deprecation official outside of the company. Can we remove that part?

Reply:
```suggestion
This feature is only available on Okteto Cloud. These instructions do not apply to, and will not work for, SaaS or Self-Hosted customers. Please us the [GitHub Actions](/cloud/preview-environments/preview-environments-github/) instructions instead to setup Preview Environments in Okteto.
```

This work?

Reply:
Ship it!